feat(i18n): add Russian locales to `git-changelog` and `enhanced-readabilities`

Ampernic opened the pull request:
Hello. We'd like to help translate plugins and documentation into our team's native language, and it might be useful for other authors as well.

As a start and for evaluation - I and @X1Z53 have added builtin localizations for `git-changelog` and `enhanced-readabilities`.

If you have any feedback or see any issues - we'd love to hear from you.

nekomeowww commented on package.json:
```diff
@@ -65,5 +65,8 @@
-  ]
+  ],
+  "dependencies": {
+    "@cspell/dict-ru_ru": "^2.2.1"
```
Should this be a `devDependencies`

nekomeowww commented:
Thank you very much for the locales support!
I am so glad we can finally push this further for the very first starting point of i18n support for Nolebase Integrations.

The rest of the code LGTM.

Ampernic commented:
> Thank you very much for the locales support! I am so glad we can finally push this further for the very first starting point of i18n support for Nolebase Integrations.
> 
> The rest of the code LGTM.

Fixed the error and corrected the linting.
Thank you
